data/core/closetheloop.md

```diff
@@ -725,38 +677,6 @@
     - utter_getstarted
     - utter_first_bot_with_rasa
 
-## Story from conversation with d75d1ac725164b3c976193eb54feb756 on November 19th 2018
-
-* greet
-    - action_greet_user
-* how_to_get_started
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-* deny
-    - action_set_onboarding
-    - slot{"onboarding":false}
-    - utter_ask_which_product
-* how_to_get_started{"product":"core"}
-    - slot{"product":"core"}
-    - utter_core_tutorial
-    - utter_anything_else
-* how_to_get_started{"product":"nlu"}
-    - slot{"product": "nlu"}
-    - utter_ask_for_nlu_specifics
-
-## Story from conversation with 7830abb04e1c49809d89b0d420443928 on November 19th 2018
-* greet
-    - action_greet_user
-* ask_whoisit
-    - action_chitchat
-* out_of_scope
-    - utter_out_of_scope
-* ask_whatspossible
-    - action_chitchat
-* how_to_get_started{"product":"nlu"}
```

Comment:
also these two?

Reply:
second is a duplicate +f ## Story from conversation with 7830abb04e1c49809d89b0d420443928 on November 19th 2018 (still there)

Reply:
first is a duplicate of ## Story from conversation with dominik on November 19th 2018